integration-tests/eks/validateResources_test.go

```diff
@@ -52,29 +57,34 @@ func TestK8s(t *testing.T) {
 	//Validating the number of pods and status
 	pods, err := ListPods(nameSpace, clientSet)
 	assert.NoError(t, err)
+	assert.Len(t, pods.Items, 3)
 	for _, pod := range pods.Items {
 		fmt.Println("pod name: " + pod.Name + " namespace:" + pod.Namespace)
+		assert.Equal(t, v1.PodRunning, pod.Status.Phase)
+		// matches
+		// - cloudwatch-agent-*
+		// - amazon-cloudwatch-observability-controller-manager-*
+		// - fluent-bit-*
+		if match, _ := regexp.MatchString(podNameRegex, pod.Name); !match {
+			assert.Fail(t, "Cluster Pods are not created correctly by Helm Charts")
```

Comment:
We are mentioning it as by Helm Charts but this validation will be used by EKS addon as well. This might be confusing.

Reply:
good catch, will update it